lib/terraforming/resource/ec2.rb

```diff
@@ -101,7 +102,9 @@ def monitoring_state(instance)
       end
 
       def instances
-        @client.describe_instances.map(&:reservations).flatten.map(&:instances).flatten.reject do |instance|
+        @client.describe_instances.map(&:reservations).flatten.map(&:instances).flatten.select do |resource|
+          @match_regex ? module_name_of(resource) =~ @match_regex : 1
+        end.reject do |instance|
```

Comment:
__[rubocop]__
* Avoid multi-line chains of blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)

<!-- Comment by [SideCI](https://sideci.com) -->